.circleci/config.yml

```diff
@@ -7,12 +7,12 @@ references:
   container_config_node14: &container_config_node14
     working_directory: ~/project/lambda-logger
     docker:
-      - image: circleci/node:14
+      - image: cimg/node:14.18.3
 
   container_config_node12: &container_config_node12
     working_directory: ~/project/lambda-logger
     docker:
-      - image: circleci/node:12
+      - image: cimg/node:12.16.3
```

Comment:
I think you can go up to 12.22 😄

Reply:
https://circleci.com/developer/images/image/cimg/node#image-name

Reply:
Thanks for pointing that out,  @GeoffThorpeFT. I've updated it :)